### PR TITLE
SRVCOM-1580: Update serverless docs structure

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1580

Applies for OCP 4.6+

Direct previews:
- https://deploy-preview-39903--osdocs.netlify.app/openshift-enterprise/latest/serverless/update/upgrading-serverless.html (new update section)
- https://deploy-preview-39903--osdocs.netlify.app/openshift-enterprise/latest/serverless/discover/serverless-understanding-eventing.html (new discover section)
- https://deploy-preview-39903--osdocs.netlify.app/openshift-enterprise/latest/serverless/monitor/cluster-logging-serverless.html (new monitor section)
- https://deploy-preview-39903--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-getting-started.html (get started section)

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -3141,10 +3140,35 @@ Topics:
     File: installing-knative-serving
   - Name: Installing Knative Eventing
     File: installing-knative-eventing
-  - Name: Upgrading the OpenShift Serverless Operator
-    File: upgrading-serverless
   - Name: Removing OpenShift Serverless
     File: removing-openshift-serverless
+- Name: Get started
```
Comment (jeana-redhat) [inline _topic_maps/_topic_map.yml:3145]: Get started appears after rel notes and install in the map now, but the "Next steps" in get started are to install the operator and look at the rel notes. Seems like it should come before those two?
image

**Review by jeana-redhat — COMMENTED** [review]
One question about the order of a couple items, but it looks like things are all working :)

File: _topic_maps/_topic_map.yml (hunk 1 above)
Comment (abrennan89) [reply _topic_maps/_topic_map.yml:3145]: Good catch. I haven't reviewed the content for all the sections yet, I'm just trying to get the buckets in line with the categories in https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCS&title=Categories+Shared+Across+Document+Sets so there's a good chance that other sections will need editing too or that this content will move. I'll remove these next steps for now though, thanks 🙂

**Pull request merged**